src/library/Cardano/CoinSelection/Fee.hs

```diff
@@ -131,35 +133,75 @@ data FeeOptions i o = FeeOptions
     } deriving Generic
 
 newtype FeeAdjustmentError
```

Comment:
This type is actually hidden from the Haddock documentation. We should give it a description so that it will be exposed.

Reply:
> This type is actually hidden from the Haddock documentation. We should give it a description so that it will be exposed.

Fixed in 34ede2fd3bc75ab0a33986b032ee575f271e0126.